crates/sui/src/sui_move/build.rs

```diff
@@ -41,7 +40,7 @@ impl Build {
             },
         )?;
         if dump_bytecode_as_base64 {
-            println!("{}", json!(pkg.get_package_base64()))
+            println!("{:?}", pkg.get_package_base64())
```

Comment:
This is a fix for the change in https://github.com/MystenLabs/sui/pull/5506. We should not revert this unless you also want to revert https://github.com/MystenLabs/sui/pull/5506

Reply:
is this a one-liner? If so I can just change it.

Reply:
yes

Reply:
Okay I put that back in.